stanford_news.module

```diff
@@ -176,6 +176,30 @@ function stanford_news_field_widget_form_alter(&$element, FormStateInterface $fo
   }
 }
 
+/**
+ * Implements hook_react_paragraphs_getfieldinfo_post_alter().
+ */
+function stanford_news_react_paragraphs_getfieldinfo_post_alter($field_element, $field_config, &$info) {
+
+  // Only alter data for the news_views view field.
+  if ($field_config->get('field_name') !== "su_news_view_field") {
+    return;
+  }
+
+  // We want to remove a few displays from the options.
+  // See stanford_news_field_widget_form_alter for the same appraoch.
+  $deny_list = ['topics_list', 'term_block', 'vertical_teaser_term_list'];
+  $vals = array_column($info['displays']['stanford_news'], 'value');
```

Comment:
can't say i've seen that function til now

Reply:
Yeah, it wasn't on top of the mind at first but I often review the array functions when it comes to working with multidimensional arrays and it turned out to be pretty darned handy.